templates/ecommerce/ecommerce.md

```diff
@@ -97,7 +97,7 @@ Now enter a name for the template instance that you are about to create. This al
 After clicking **Next Step**, you will see the template builder. Fill in all needed credentials and perform the required OAuth authorizations. 
 
 **Important:** Make sure to follow all the steps very carefully to prevent the newly created flow from failing because of any user authorization problems. 
-If you are struggling with this part, go to the section [Authorizing Destinations](/templates/ecommerce/authorizing-destinations/) below.
+If you are struggling with this part, go to the section [Authorizing Data Destinations](/templates/ecommerce/#authorizing-data-destinations/) below.
```

Comment:
Why is the slash here? (#)

Reply:
to neni slash :-)
# tam byt musí, protože to odkazuje na kapitolu v rámci toho samýho článku

Reply:
minule jsem to prehlidla a nikdo jinej na to od ty doby neklikl, aby zjistil, ze ty odkazy nefungujou

Reply:
aha, dobře, rozumím, to jsem nevěděla :-)